aidies-project/db/migrations/20200504040340-create-user.js

```diff
@@ -42,6 +42,9 @@ module.exports = {
       facebook: {
         type: Sequelize.STRING,
       },
+      tokenId: {
+        type: Sequelize.STRING,
+      },
```

Comment:
We can actually remove the `tokenId` because our cookie will take care of giving us the JWT token in the backend!